### PR TITLE
Update fa.json (pr.json)

### PR DESCRIPTION
**Pull request by miko-github**
# Hi

## you may want to rename
i think, you should **rename** this file to **Persian** (pr.json), because we not have **_Farsi_**,
actually **Farsi**, is Arabic name for Persian language with **Dari** branch (in Iran, Afghanistan, ..) and other branch is Pashto, Urdo, Tajik (or Tajikistan),

## Why
because **Parsi** is Persian name for **Persian** language, but **Arabic** not have **P** sound, then, say **Farsi** :wink: 

## TIps
in Persian, we have some word that have Arabic nouns also, example,
نام => name (sound : Nām)
but, we also Arabic word for this, but not common, 
اسم => name (sound : Esm)

thank :heart:

### CODE REVIEW

**Comment by miko-github** [comment]
Ops, forgive me, i forgot remove some comments at front of properties, when i translated them :smile: 
sorry :+1:

**Comment by PalmerAL** [comment]
I think I understand what you mean about "Persian" vs "Farsi", but the language names are chosen by Chromium, and we have to use the same ones that they do: https://www.electronjs.org/docs/api/locales. They use `fa` for Persian, so we need to as well.

File: localization/languages/fa.json [hunk 1]
```diff
@@ -5,25 +5,25 @@
...
+    "openImageInNewPrivateTab": "نمایش تصویر در برگه خصوصی جدید",
+    "saveImageAs": "ذخیره تصویر به عنوان",
     "searchWith": "جستجو با %s", //%s will be replaced with the name of the current search engine
-    "copyLink": "کپی لینک",
-    "copyEmailAddress": null, //missing translation
-    "selectAll": null, // Missing Translation
-    "undo": null, // Missing Translation
+    "copyLink": "کپی پیوند",
+    "copyEmailAddress": "کپی رایانامه",
+    "selectAll": "انتخاب همه", // this is farsi, but in persian we say گزینش همه
```
Comment (PalmerAL) [inline localization/languages/fa.json:20]: What translation does Chrome use here? We can probably use the same one.

File: localization/languages/fa.json [hunk 2]
```diff
@@ -144,16 +144,18 @@
     "settingsCustomBangsRedirect": null, // missing translation
     "settingsCustomizeFiltersLink": null, //missing translation
     "settingsAppearanceHeading": "ظاهر",
-    "settingsEnableDarkMode": null, //"Enable dark mode:" missing translation
-    "settingsDarkModeNever": null, //"Never" missing translation
+    "settingsEnableDarkMode": "پوسته تاریک روشن", //"Enable dark mode:" missing translation
+    "settingsDarkModeNever": "پوسته تاریک هرگز", //"Never" missing translation
```
Comment (PalmerAL) [inline localization/languages/fa.json:148]: Can you remove the comments?

File: localization/languages/fa.json [hunk 3]
```diff
@@ -144,16 +144,18 @@
...
+    "settingsSiteThemeToggle": "تغییر پوسته", //missing translation
     "settingsAdditionalFeaturesHeading": "امکانات اضافه",
     "settingsUserscriptsToggle": "فعال کردن اسکریپت های کاربر",
-    "settingsShowDividerToggle": null, // Missing translation
-    "settingsSeparateTitlebarToggle": null, //missing translation
-    "settingsAutoplayToggle": null, //missing translation 
+    // both Divider and Separater is same in persian => جداساز
+    // but i didn't know use for which one,
+    "settingsShowDividerToggle": null, // Missing translation, may نمایش جداساز
+    "settingsSeparateTitlebarToggle": null, //missing translation, may جداساز تیتر
```
Comment (PalmerAL) [inline localization/languages/fa.json:157]: "Separate" is an adjective here - "Use separate title bar" means that the title bar is a separate piece from the tab bar: https://github.com/minbrowser/min/pull/977#discussion_r414918201

Is there a way to translate that?

File: localization/languages/fa.json (hunk 2 above)
Comment (miko-github) [reply localization/languages/fa.json:148]: yes, i try for this :hugs:

File: localization/languages/fa.json (hunk 3 above)
Comment (miko-github) [reply localization/languages/fa.json:157]: thanks, i know what meant separate, but in Persian we just have one phrase for these, may change verb of this
separate => جداکننده
divider => جداساز
but this haven't different :smile:  :hugs:

File: localization/languages/fa.json (hunk 1 above)
Comment (miko-github) [reply localization/languages/fa.json:20]: ok, i searching for them in my friends :+1:

**Comment by miko-github** [comment]
so, if i create build for ubuntu, do you merge it? :thinking: 
so, this browser only support mac and window and don't support Ubuntu!!

File: localization/languages/fa.json [hunk 4]
```diff
@@ -144,16 +144,18 @@
...
-    "settingsEnableDarkMode": null, //"Enable dark mode:" missing translation
-    "settingsDarkModeNever": null, //"Never" missing translation
+    "settingsEnableDarkMode": "پوسته تاریک روشن", //"Enable dark mode:" missing translation
+    "settingsDarkModeNever": "پوسته تاریک هرگز", //"Never" missing translation
     "settingsDarkModeNight": null, //"At night" missing translation
-    "settingsDarkModeAlways": null, //"Always" missing translation
-    "settingsSiteThemeToggle": null, //missing translation
+    "settingsDarkModeAlways": "پوسته تاریک برای همیشه", //"Always" missing translation
+    "settingsSiteThemeToggle": "تغییر پوسته", //missing translation
     "settingsAdditionalFeaturesHeading": "امکانات اضافه",
```
Comment (miko-github) [inline localization/languages/fa.json:152]: - [ ] change to امکانات افزوده

File: localization/languages/fa.json [hunk 5]
```diff
@@ -144,16 +144,18 @@
...
     "settingsUserscriptsToggle": "فعال کردن اسکریپت های کاربر",
-    "settingsShowDividerToggle": null, // Missing translation
-    "settingsSeparateTitlebarToggle": null, //missing translation
-    "settingsAutoplayToggle": null, //missing translation 
+    // both Divider and Separater is same in persian => جداساز
```
Comment (miko-github) [inline localization/languages/fa.json:154]: - [ ] update Maoudi suggest translated

File: localization/languages/fa.json [hunk 6]
```diff
@@ -203,9 +205,9 @@
     "appMenuInspectBrowser": "وارسی مرورگر",
     "appMenuInspectPage": "وارسی صفحه",
     "appMenuWindow": "پنجره",
-    "appMenuMinimize": "کوچک نمایی",
+    "appMenuMinimize": "کوچک نمایی", // in persian we say کمینه but this also true
     "appMenuClose": "بستن",
-    "appMenuAlwaysOnTop": null, //missing translation
+    "appMenuAlwaysOnTop": "قرار دادن در بالا", //missing translation
```
Comment (miko-github) [inline localization/languages/fa.json:210]: - [ ] همیشه در بالا || نمایش همیشگی

File: localization/languages/fa.json [hunk 7]
```diff
@@ -203,9 +205,9 @@
     "appMenuInspectBrowser": "وارسی مرورگر",
     "appMenuInspectPage": "وارسی صفحه",
     "appMenuWindow": "پنجره",
-    "appMenuMinimize": "کوچک نمایی",
+    "appMenuMinimize": "کوچک نمایی", // in persian we say کمینه but this also true
```
Comment (miko-github) [inline localization/languages/fa.json:208]: - [ ] کمینه نمایی || کمینه گری || کمینه

File: localization/languages/fa.json [hunk 8]
```diff
@@ -203,9 +205,9 @@
     "appMenuInspectBrowser": "وارسی مرورگر",
     "appMenuInspectPage": "وارسی صفحه",
```
Comment (miko-github) [inline localization/languages/fa.json:206]: - [ ] وارسی برگه

File: localization/languages/fa.json [hunk 9]
```diff
@@ -168,9 +170,9 @@
...
     "settingsProxyHeading": null, //missing translation
-    "settingsNoProxy": null, //missing translation
-    "settingsManualProxy": null, //missing translation
-    "settingsAutomaticProxy": null, //missing translation
+    "settingsNoProxy": "بدون پروکسی",
+    "settingsManualProxy": "پروکسی دستی",
+    "settingsAutomaticProxy": "پروکسی خودکار",
     "settingsProxyRules": null, //missing translation
     "settingsProxyBypassRules": null, //missing translation
     "settingsProxyConfigurationURL": null, //missing translation
```
Comment (miko-github) [inline localization/languages/fa.json:178]: - [ ] قوانین پروکسی

File: localization/languages/fa.json [hunk 10]
```diff
@@ -144,16 +144,18 @@
...
     "settingsCustomizeFiltersLink": null, //missing translation
     "settingsAppearanceHeading": "ظاهر",
-    "settingsEnableDarkMode": null, //"Enable dark mode:" missing translation
-    "settingsDarkModeNever": null, //"Never" missing translation
+    "settingsEnableDarkMode": "پوسته تاریک روشن", //"Enable dark mode:" missing translation
+    "settingsDarkModeNever": "پوسته تاریک هرگز", //"Never" missing translation
     "settingsDarkModeNight": null, //"At night" missing translation
-    "settingsDarkModeAlways": null, //"Always" missing translation
-    "settingsSiteThemeToggle": null, //missing translation
+    "settingsDarkModeAlways": "پوسته تاریک برای همیشه", //"Always" missing translation
```
Comment (miko-github) [inline localization/languages/fa.json:150]: - [ ] remove همیشه

**Comment by PalmerAL** [comment]
> so, if i create build for ubuntu, do you merge it? 🤔
> or your do this?
> so, this browser only support mac and window and don't support Ubuntu!!

This repository contains the code to build the browser. Every month or two, I make a new build for each operating system and post it on the releases page. So you don't need to build anything.

**Comment by miko-github** [comment]
Update finish :heart:

**Comment by PalmerAL** [comment]
OK, thank you! Everything looks good.

**Pull request merged**